src/OrleansManager/Program.cs

```diff
@@ -10,7 +10,6 @@ namespace OrleansManager
     class Program
     {
         private static IManagementGrain systemManagement;
-        const int RETRIES = 3;
```

Comment:
retry was counting down, hop count is counting up. It always starts at zero. I made it an optional argument that can be omitted on the first call to that method. The first caller is always a local call so it does not even know that there may be recursion later on, so I though it was unnecessary to pass 0 in all those calls.
```csharp
Task DeleteGrainAsync(GrainId grainId, int hopCount = 0);
```